pytorch_lightning/plugins/precision/mixed.py

```diff
@@ -23,4 +23,7 @@ class MixedPrecisionPlugin(PrecisionPlugin):
     """Base Class for mixed precision."""
 
     backend: "AMPType"
```

Comment:
I think we can get rid of this now

Reply:
this is deprecated in https://github.com/PyTorchLightning/pytorch-lightning/pull/12069